IsraelHiking.Web/src/application/services/share-urls.service.ts

```diff
@@ -168,12 +168,7 @@ export class ShareUrlsService {
     }
 
     public setShareUrl(shareUrl: ShareUrl) {
-        this.ngRedux.dispatch(new SetShareUrlAction({
-            shareUrl
-        }));
-        if (shareUrl != null) {
-            this.ngRedux.dispatch(new SetPannedAction({ pannedTimestamp: new Date() }));
-        }
+        this.store.dispatch(new SetShareUrlAction(shareUrl));
```

Comment:
This was an addition to a bug I fixed, need to find a way to check that this is not needed.